### ISSUE TITLE
numlock layer colormap not showing in editor

### ISSUE THREAD

**Issue by qayshp** [keyboardio/Chrysalis#319]
**Describe the bug**
When enabling numlock layer, my keyboard colors change accordingly, which is great! I don't, however, see the colormap for this in the colormap editor for either the Read Only or custom numlock layers.

**To Reproduce**
1. Colormap editor
2. Click on a numlock layer
3. notice that the colormap is still all the default

**Expected behavior**
See the colormap showing that the right half numpad pattern keys would be red.

**Screenshots**
![image](https://user-images.githubusercontent.com/714080/52602892-3634c980-2e19-11e9-8779-9e4db80836ca.png)
![image](https://user-images.githubusercontent.com/714080/52602922-4fd61100-2e19-11e9-9f0d-a30138454c51.png)

Nothing special showing in the log, but let me know if there's anything you need from it.

**Desktop (please complete the following information):**
 - OS: macOS 10.14.3
 - Chrysalis Version: 0.4.0+53

**Additional context**
N/A

**Comment by algernon** [comment]
This is kind of "expected" behaviour. The colormap editor works with the colors provided by the `Colormap` plugin. Any other plugin that changes colors one way or the other, are not reflected in the editor. We currently do not have any plans to change this, but this should be much better documented to avoid this kind of surprises and confusion.

**Comment by mattmc3** [comment]
What if we want the Chrysalis color map to take precedence of other plugins? Any way to do that? I'm experiencing a similar issue where I am using the butterfly key to switch from QWERTY to lock to a Colemak layer and all my remaps light up red which is really not ideal. Any fn key layer colors also don't show.

**Comment by nickhandel** [comment]
I've just discovered this as well - out-of-the-box, I can set the Layer 1 colours to anything I like and it doesn't take effect.  From a dumb end user point-of-view, is there no way of changing this without getting into customising the firmware?

**Comment by qayshp** [comment]
I understand that showing what keys/colors a plugin changes would be overkill.

Having a way to toggle stock firmware plugins without building a sketch would be a nice-to-have.

**Comment by alerque** [comment]
My Model 100 arrived today. I'm pretty stoked, they keyboard is pretty nice. The first issue I ran into was the reasonably well documented relating to the stock firmware not reporting a version and thus needing a modified procedure to update.

The second issue I ran into was this one. I hacked on my layouts for a long time trying to bring them up to speed with my previous Kinesis setup. Then I started color coding keys into groups. Everything went fine except the numlock layer. No matter what I tried the colors just would not apply. Sometimes 1 key would cooperate, usually the keys I defined had colors (as opposed to being undefined) but never would they obey my settings.

Finally reading through this issue I found a workaround: copy layer 1 to an unused layer, then change the numlock modified to move to that layer. Now the colormaps work just fine as well as the keys.

This is unexpected and still kind of inexplicable. Why is layer 1 magic in a way that allows you to program the layer as usual but not the colors?
